packages/material-ui/src/OutlinedInput/OutlinedInput.js

```diff
@@ -60,8 +60,8 @@ export const styles = theme => {
     },
     /* Styles applied to the `input` element if `margin="dense"`. */
     inputMarginDense: {
-      paddingTop: 15,
-      paddingBottom: 15,
+      paddingTop: 8,
```

Comment:
The dense textfield height should be 40px (a divisor of 8), both for the outlined and filled variant. It's the value I get measuring the specification images. How did you pick these values?

Reply:
Oh yeah, right. I'll try using the test suite und update the Filled and Outlined Input.